features/fetch.yml

```diff
@@ -1,7 +1,13 @@
-name: Fetch (initial support)
```

Comment:
@ddbeck I changed the name here. I feel like we'll have very few "initial support" features left in the long term, but am not sure what the name guideline should be.

Reply:
Good call on renaming this. Thank you!